### PR TITLE
Project owner permissions

### PR DESCRIPTION
**Pull request by CjMoore**
- adds permissions for project owner role
- narrows regular user permissions to only include logging out in sessions
- rspec 54-0

### CODE REVIEW

File: app/services/permission.rb [hunk 1]
```diff
@@ -32,8 +34,17 @@ def guest_user_permissions
...
-    return true if controller == "sessions"
-    return true if controller == "users"
+    return true if controller == "sessions" && action.in?(["destroy"])
+    return true if controller == "users" && action.in?(["show"])
+    return true if controller == "projects" && action.in?(["index", "show", "new", "create"])
+    return true if controller == "comments" && action.in?(["create"])
+    return true if controller == "user_funded_projects" && action.in?(["new"])
+  end
+
+  def project_owner_permissions
```
Comment (cews7) [inline app/services/permission.rb]: @CjMoore are these supposed to have the same permissions?

Comment (CjMoore) [inline app/services/permission.rb]: At this point we don't have any controllers or controller actions that are exclusive to a project owner. When we implement edit project and disable project we will have to add those permissions to project owner. @cews7

Comment (cews7) [inline app/services/permission.rb]: @CjMoore shouldn't project owners have the ability to disable a project?

Comment (CjMoore) [inline app/services/permission.rb]: Yes. But we don't currently have that functionality. When we add that functionality, we'll have to add that permission. @cews7

**Pull request merged**